packages/contract/README.md

```diff
@@ -310,6 +310,52 @@ const val = instance.getValue();
 // since the contract returns that value.
 ```
 
+#### Overloaded functions
+
+You may find yourself in the situation of having a Truffle contract object that has multiple functions with the same name.
+You can call these "overloaded functions" just like you would a normal contract function. Truffle contract instances
```

Comment:
```suggestion
You can invoke these overloaded functions just like you would a normal contract function. Truffle contract instances
```

Reply:
I like the change to `invoke` but I prefer to leave overloaded functions to be emphasized in some way to make it clear that this is terminology for people unfamiliar with it.

Reply:
My own editing comment here is that if you're going to call out something, it should be just the word "overloaded", not the whole noun phrase.